src/lava/lib/optimization/solvers/generic/solver.py

```diff
@@ -61,6 +71,11 @@
     from lava.lib.optimization.solvers.generic.read_gate.ncmodels import (
         ReadGateCModel,
     )
+
+    from lava.lib.optimization.solvers.generic.qp.ncmodels import (

```

Comment:
why does the name specify L2? We cannot implement this models on L1 anyway.

Reply:
no, but maybe we can in L3